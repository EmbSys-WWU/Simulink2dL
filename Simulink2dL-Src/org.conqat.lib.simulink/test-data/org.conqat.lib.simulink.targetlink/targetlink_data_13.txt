{'logdata',{'plotchannels',[]},'output',{'variable','DPFRc_prevActvRgnrtnStatusSt'},'state',{'max',30,'min',0,'type','uint8','scaling','./LocalScaling'},'date',734875.60776526621}