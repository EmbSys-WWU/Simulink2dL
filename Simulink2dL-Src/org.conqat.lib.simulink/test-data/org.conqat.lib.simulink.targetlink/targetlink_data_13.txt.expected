struct (
  'logdata' -> struct (
    'plotchannels' -> '[]'
  )
  'output' -> struct (
    'variable' -> 'DPFRc_prevActvRgnrtnStatusSt'
  )
  'state' -> struct (
    'max' -> '30'
    'min' -> '0'
    'scaling' -> './LocalScaling'
    'type' -> 'uint8'
  )
  'date' -> '734875.60776526621'
)